service/internal/pipelines/pipelines.go

```diff
@@ -304,6 +304,23 @@ func Build(ctx context.Context, settings component.TelemetrySettings, buildInfo
 	return exps, nil
 }
 
+func logStabilityMessage(logger *zap.Logger, sl component.StabilityLevel) {
+	switch sl {
+	case component.StabilityLevelDeprecated:
+		logger.Info("Component has been deprecated and will be removed in future releases.", zap.String(components.ZapStabilityKey, sl.String()))
+	case component.StabilityLevelUnmaintained:
+		logger.Info("Component is unmaintained and actively looking for contributors.", zap.String(components.ZapStabilityKey, sl.String()))
+	case component.StabilityLevelInDevelopment:
+		logger.Info("Component is under development.", zap.String(components.ZapStabilityKey, sl.String()))
+	case component.StabilityLevelAlpha:
+	case component.StabilityLevelBeta:
+	case component.StabilityLevelStable:
```

Comment:
I think L315 to L318 should be changed to

```
case component.StabilityLevelAlpha, component.StabilityLevelBeta, component.StabilityLevelStable:
   logger.Debug("Stability level", zap.String(components.ZapStabilityKey, sl.String()))
```
or

```
	case component.StabilityLevelAlpha:
             fallthrough
	case component.StabilityLevelBeta:
             fallthrough
	case component.StabilityLevelStable:
             logger.Debug("Stability level", zap.String(components.ZapStabilityKey, sl.String()))
```

Reply:
Fixed